site/partials/navLinks.html

```diff
@@ -20,3 +20,14 @@
     About
   </a>
 </div>
+
+<div class="nav-right">
+  <div class="field nav-item">
+    <p class="control has-icon">
+      <input id="docSearchTextbox" class="input" type="text" placeholder="Search...">
+      <span class="icon is-small">
+        <i class="fa fa-search"></i>
+      </span>
+    </p>
+  </div>
+</div>
```

Comment:
Actually it was already set up in VSCode, but I probably manually removed the line